src/main/scala/karazin/scala/users/group/week2/homework/Homework.scala

```diff
@@ -55,8 +60,14 @@ object Homework:
 
     private lazy val g = gcd(abs(x), y)
 
-    override def equals(other: Any): Boolean = ???
+    override def equals(other: Any): Boolean =
```

Comment:
Contract violation. Please review https://www.baeldung.com/java-equals-hashcode-contracts
If you overload `equals` then `hashCode` should be overloaded as well.